app/src/main/res/values/styles.xml

```diff
@@ -19,8 +19,6 @@
         <item name="android:textColor">@color/primary_text_color</item>
         <item name="android:statusBarColor">@color/primary_dark_color</item>
         <item name="android:navigationBarColor">@android:color/transparent</item>
-        <item name="android:windowLightStatusBar" tools:targetApi="m">@bool/light_status_bar</item>
```

Comment:
`#ytho`

Reply:
We have seperate styles for v23, v27 and v29. This is present in all of those.

Reply:
:+1: